src/tests/multihost/alltests/test_default_debug_level.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+import re
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.defaultdebuglevel
+class TestDefaultDebugLevel(object):
+    """ Check sssd default debug level """
+    @pytest.mark.tier1
+    def test_0001_check_default_debug_level(self, multihost, backupsssdconf):
+        """
+        @Title: IDM-SSSD-TC: Check default debug level by authenticating the
+        user
+        """
+        section = "domain/%s" % ds_instance_name
+        domain_params = {'debug_level': ''}
+        tools = sssdTools(multihost.client[0])
+        tools.sssd_conf(section, domain_params, action='delete')
+        tools.clear_sssd_cache()
+        user = 'foo1@%s' % ds_instance_name
+        client = pexpect_ssh(multihost.client[0].sys_hostname, user,
+                             'Secret123', debug=False)
+        client.login(login_timeout=30, sync_multiplier=5,
+                     auto_prompt_reset=False)
+        log_list = ['sssd.log', 'sssd_example1.log', 'sssd_nss.log',
+                    'sssd_pam.log', 'sssd_implicit_files.log']
+        for log in log_list:
+            log = '/var/log/sssd/%s' % log
```

Comment:
I would propose to avoid redefinition of var `log`. `log_filename` would be more readable, IMO...